### PR TITLE
Tools: Don't use avalon tools code

### PR DESCRIPTION
**Pull request by iLLiCiTiT**
## Brief description
Planned movement of code requires to cleanup code in openpype first. One change is that 

## Description
Changed imports to not use lib functions and models from `avalon.tools` but from `openpype.tools.utils`.

## Changes
- changed imports
- moved pyblish gui discover logic into openpype

## Testing notes:
Nothing should change. Affected places:
- Standalone publishing should show publish tool
- Scene inventory should display versions correctly
- Maya should have pyblish icon in menu
- mayalookassigner should work before
- workfiles tool should work as before

### CODE REVIEW

**Comment by mkolar** [comment]
Task linked: [OP-2814 Use openpype tools only](https://app.clickup.com/t/6658547/OP-2814)

File: openpype/tools/mayalookassigner/widgets.py [hunk 1]
```diff
@@ -4,8 +4,11 @@
 from Qt import QtWidgets, QtCore
 
 # TODO: expose this better in avalon core
```
Comment (BigRoy) [inline openpype/tools/mayalookassigner/widgets.py:6]: I guess this comment can also be removed? :)

Comment (mkolar) [inline openpype/tools/mayalookassigner/widgets.py:6]:
```suggestion
```

**Review by mkolar — APPROVED** [review]
Seem to work all fine for me

**Comment by BigRoy** [comment]
Should we also refactor this comment?

- https://github.com/pypeclub/OpenPype/blob/enhancement/OP-2814_Use-openpype-tools-only/openpype/hosts/harmony/api/lib.py#L382

**Comment by iLLiCiTiT** [comment]
> Should we also refactor this comment?

I've asked and the harmony plugin is installed automatically. I've removed the backwards compatibility.

**Pull request merged**